cmd/src/config.json

```diff
@@ -0,0 +1 @@
+{"accessToken": "d2086ad0b4f2464647fa38fdc3dedd54f3361dc5", "endpoint": "http://localhost:7080"}
```

Comment:
This shouldn't be in your PR

Reply:
@attfarhan 	make sure to revoke that token asap

Reply:
This was for my local instance. Revoked it though.